src/ducks/reimbursements/HealthReimbursements.jsx

```diff
@@ -62,25 +62,37 @@ class DumbHealthReimbursements extends Component {
             {t('Reimbursements.awaiting')}
           </Title>
         </Padded>
-        <TransactionsWithSelection
-          transactions={pendingTransactions}
-          brands={this.props.brands}
-          urls={this.props.urls}
-          withScroll={false}
-          className={styles.HealthReimbursements__transactionsList}
-        />
+        {pendingTransactions ? (
+          <TransactionsWithSelection
+            transactions={pendingTransactions}
+            brands={this.props.brands}
+            urls={this.props.urls}
+            withScroll={false}
+            className={styles.HealthReimbursements__transactionsList}
+          />
+        ) : (
+          <Padded className="u-pv-0">
```

Comment:
Why do we need a u-pv-0 ? Because the `p` has a padding ?

Reply:
Yeah. Maybe we should add `horizontal` and `vertical` props to `Padded`, so we don't have to use utilities when we need padding in only one direction?